package.json

```diff
@@ -7,60 +7,60 @@
     "url": "https://github.com/mozilla/activity-stream/issues"
   },
   "dependencies": {
-    "prop-types": "15.6.0",
+    "prop-types": "15.6.1",
     "react": "16.2.0",
     "react-dom": "16.2.0",
     "react-intl": "2.4.0",
     "react-redux": "5.0.6",
     "redux": "3.6.0"
   },
   "devDependencies": {
-    "@octokit/rest": "14.0.8",
-    "babel-core": "6.26.0",
-    "babel-loader": "7.1.2",
+    "@octokit/rest": "15.3.0",
+    "babel-core": "6.26.3",
+    "babel-loader": "7.1.4",
     "babel-plugin-jsm-to-commonjs": "0.4.0",
     "babel-plugin-jsm-to-esmodules": "0.4.0",
     "babel-plugin-transform-async-to-module-method": "6.24.1",
-    "babel-plugin-transform-es2015-modules-commonjs": "6.26.0",
+    "babel-plugin-transform-es2015-modules-commonjs": "6.26.2",
     "babel-preset-react": "6.24.1",
     "chai": "4.1.2",
     "co-task": "1.0.0",
     "cpx": "1.5.0",
     "enzyme": "3.3.0",
     "enzyme-adapter-react-16": "1.1.1",
-    "eslint": "4.17.0",
-    "eslint-plugin-import": "2.8.0",
+    "eslint": "4.19.1",
+    "eslint-plugin-import": "2.11.0",
     "eslint-plugin-json": "1.2.0",
-    "eslint-plugin-mozilla": "0.8.1",
-    "eslint-plugin-no-unsanitized": "2.0.2",
-    "eslint-plugin-promise": "3.6.0",
-    "eslint-plugin-react": "7.6.1",
-    "eslint-watch": "3.1.3",
+    "eslint-plugin-mozilla": "0.11.0",
+    "eslint-plugin-no-unsanitized": "3.0.0",
+    "eslint-plugin-promise": "3.7.0",
+    "eslint-plugin-react": "7.7.0",
+    "eslint-watch": "3.1.4",
     "husky": "0.14.3",
     "istanbul-instrumenter-loader": "0.2.0",
     "joi-browser": "13.0.1",
-    "karma": "1.7.1",
+    "karma": "2.0.2",
```

Comment:
The last time we were greening dependencies karma  2.0.0 had problems with source maps (see https://bugzilla.mozilla.org/show_bug.cgi?id=1441278) and we needed to downgrade it. Let me confirm this is still the case, it may be fixed now

Reply:
Yup, I confirmed it yesterday but good to make sure it wasn't just some local configuration of mine: https://bugzilla.mozilla.org/show_bug.cgi?id=1441278#c9

Reply:
Excellent! it is indeed working 👍